### PR TITLE
style(celery): Formatting consistency on crontab defs

### PR DESCRIPTION
**Pull request by evanpurkhiser**
Just trying to keep things consistent here

### CODE REVIEW

File: src/sentry/conf/server.py [hunk 1]
```diff
@@ -1010,16 +1013,19 @@ def SOCIAL_AUTH_DEFAULT_USERNAME() -> str:
     },
     "flush-buffers": {
         "task": "sentry.tasks.process_buffer.process_pending",
+        # Run every 10 minutes
```
Comment (vartec) [inline src/sentry/conf/server.py:1016]:
```suggestion
        # Run every 10 seconds
```

File: src/sentry/conf/server.py [hunk 2]
```diff
@@ -1010,16 +1013,19 @@ def SOCIAL_AUTH_DEFAULT_USERNAME() -> str:
...
         "schedule": timedelta(seconds=10),
         "options": {"expires": 10, "queue": "buffers.process_pending"},
     },
     "sync-options": {
         "task": "sentry.tasks.options.sync_options",
+        # Run every 10 minutes
```
Comment (vartec) [inline src/sentry/conf/server.py:1022]:
```suggestion
        # Run every 10 seconds
```

File: src/sentry/conf/server.py [hunk 3]
```diff
@@ -1010,16 +1013,19 @@ def SOCIAL_AUTH_DEFAULT_USERNAME() -> str:
...
         "schedule": timedelta(seconds=10),
         "options": {"expires": 10, "queue": "options"},
     },
     "schedule-digests": {
         "task": "sentry.tasks.digests.schedule_digests",
+        # Run every 30 minutes
```
Comment (vartec) [inline src/sentry/conf/server.py:1028]:
```suggestion
        # Run every 30 seconds
```

Comment (evanpurkhiser) [reply src/sentry/conf/server.py:1028]: Thanhks for catching these

**Review by vartec — APPROVED** [review]

**Pull request merged**